package/ripgrep/package

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(ripgrep)
+pkgdesc="Modern grep for recursive regex pattern searching"
+url=https://github.com/BurntSushi/ripgrep
+pkgver=13.0.0-1
+timestamp=2021-06-12T10:54Z
+section="utils"
+maintainer="Mattéo Delabre <spam@delab.re>"
```

Comment:
@matteodelabre are you okay with being the maintainer instead of @gbyl ?

Reply:
My apologies, I left that from the template I copied. I will fix it now.